packages/videoCutting/FastAPI.py

```diff
@@ -0,0 +1,36 @@
+import subprocess
+import uvicorn
+from fastapi import FastAPI
+from dotenv import load_dotenv, set_key
+
+app = FastAPI()
+
+dotenv_path = ".env"
+load_dotenv(dotenv_path)
+
+
+# api 範例: http://localhost:8000/processimg/?param1=./input_folder&param2=./output_folder
+@app.get("/processimg/")
+def process_image(param1: str, param2: str): # 接受兩個參數，分別代表輸入圖片路徑跟輸出圖片路徑

```

Comment:
Sorry to review this now.
suggestion: Can we make it more scalable so we can process an image at a time and to prevent security issues with query params maybe we can use POST instead, thanks
```suggestion
# api 範例: http://localhost:8000/processimg
@app.post("/processimg/")
def process_image(req): 
    input_image_path = body['input_img_path']
    output_image_path = body['output_img_path']
```
again, sorry for the late review and not saying the requirements when you're implementing🙇‍♂️🙏